src/main/java/io/xpring/payid/PayID.java

```diff
@@ -13,6 +13,8 @@
  * A standardized identifier for payment accounts.
  *
  * @see "https://github.com/xpring-eng/rfcs/blob/master/TBD.md"
+ *
+ * @deprecated Please use the idiomatically named `PayId` interface instead.
```

Comment:
Why doesn't this class have an `@Deprecated` decorator, but others such as `PayIDComponents` and `AbstractPayID` do?

Reply:
Purely an oversight.